src/elf/reloc.rs

```diff
@@ -191,7 +189,9 @@ macro_rules! elf_rela_std_impl { ($size:ident, $isize:ty) => {
                 let count = size / SIZEOF_RELA;
                 let mut relocs = vec![Rela::default(); count];
                 fd.seek(Start(offset as u64))?;
-                fd.read(relocs.as_mut_bytes())?;
+                unsafe {
+                    fd.read(plain::as_mut_bytes(&mut *relocs))?;
```

Comment:
The unsafety is technically in the as_mut_bytes so this is slightly over estimated, but I don't care. 

But why is as_mut_bytes unsafe now ?